Flyデプロイがイメージ not foundで失敗する

#721 で私が壊しました

## 💡 Summary

- mainブランチのflyデプロイが失敗している
- エラー「そんなOSないよ」

```console
#7 [internal] load metadata for quay.io/evl.ms/fullstaq-ruby:3.3.1-bullseye-jemalloc-slim
#7 ERROR: quay.io/evl.ms/fullstaq-ruby:3.3.1-bullseye-jemalloc-slim: not found
------
 > [internal] load metadata for quay.io/evl.ms/fullstaq-ruby:3.3.1-bullseye-jemalloc-slim:
------
Error: failed to fetch an image or build from source: error building: failed to solve: quay.io/evl.ms/fullstaq-ruby:3.3.1-bullseye-jemalloc-slim: not found
Error: Process completed with exit code 1.
```

## 原因

- OSバリアントの名前の順番を間違えた
  - 誤：3.3.1-bullseye-jemalloc-slim
  - 正：3.3.1-jemalloc-bullseye-slim

--- a/fly.Dockerfile
+++ b/fly.Dockerfile
@@ -18,9 +18,8 @@
 # performance.
 
 ARG RUBY_VERSION=3.3.1
-ARG VARIANT=jemalloc-slim
-ARG OS_VERSION=bullseye
-FROM quay.io/evl.ms/fullstaq-ruby:${RUBY_VERSION}-${OS_VERSION}-${VARIANT} as base
+ARG VARIANT=jemalloc-bullseye-slim
+FROM quay.io/evl.ms/fullstaq-ruby:${RUBY_VERSION}-${VARIANT} as base
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 

